src/ActorMultiVertex.cpp

```diff
@@ -1032,10 +1057,10 @@ class LunaActorMultiVertex: public Luna<ActorMultiVertex>
 		{
 			luaL_error(L, "The texture must be set before adding states.");
 		}
-		const float width_pix= tex->GetImageToTexCoordsRatioX();
-		const float height_pix= tex->GetImageToTexCoordsRatioY();
-		const float width_ratio= 1.0f / tex->GetImageToTexCoordsRatioX();
-		const float height_ratio= 1.0f / tex->GetImageToTexCoordsRatioY();
+		const float width_pix = tex ? tex->GetImageToTexCoordsRatioX() : 0;
```

Comment:
Same with these changes. They're not needed because of the luaL_error check above.